src/System Application/App/Email/src/Connector/DefaultEmailConnectorv2.Codeunit.al

```diff
@@ -0,0 +1,59 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Email;
+
+codeunit 8998 "Default Email Connector v2" implements "Email Connector v2"
```

Comment:
Is v2 the best we can do? Is there some descriptor of what functionality this adds we could use instead?

Reply:
It is the best I can do for now. Added information into the summary docs.